ote/src/cljs/ote/views/transport_operator_ytj.cljs

```diff
@@ -145,19 +161,28 @@
 
       (when (and response-ok? (not ytj-company-names-found?))
         {:name :msg-no-aux-names-for-business-id
-         :label (tr [:common-texts :no-aux-names-for-business-id])
+         :label (tr [:organization-page :no-aux-names-for-business-id])
          :type :text-label
          :max-length 128})
 
       {:name       :msg-business-id-contact-details
        :label      (if ytj-company-names-found?
-                     (tr [:common-texts :contact-details-plural])
-                     (tr [:common-texts :contact-details])
+                     (tr [:organization-page :contact-details-plural])
+                     (tr [:organization-page :contact-details])
                      )
        :type       :text-label
        :max-length 128
        :h-style    :h3}
 
+      (when response-ok?
+        {:name          :help-operator-contact-details
+         :type          :info-toggle
+         :label         (tr [:common-texts :instructions])
+         :body          [:div
+                         (tr [:organization-page :help-operator-contact-entry])
+                         [uicommon/extended-help-link "https://www.ytj.fi/index/ilmoittaminen/muutosilmoitus.html" "Tietojen muutosilmoitus Yritys- ja yhteisötietojärjestelmässä (YTJ)"]]
```

Comment:
You need to pass this url based on users language. In english: https://www.ytj.fi/en/index/notifications/notificationsofchanges.html